gluon-activations/gluon-activations.md

```diff
@@ -0,0 +1,239 @@
+
+## MXNet Activations Tutorial
+
+Deep neural networks encode a nonlinear, highly parametrized function from input to outputs. The nonlinearities that allow neural networks to be maximally expressive are referred to as activation functions. Over the course of the development of neural networks, several nonlinear activation functions have been introduced to make gradient-based deep learning tractable. 
+
+If you are looking to answer the question, which activation function should I use for my neural network model you should probably go with *ReLU*. Unless, you're trying to implement something like a gating mechanism, like in LSTMs or GRU cells, then you should opt for sigmoid and/or tanh in those cells. However, if you have a working model architecture and you're trying to improve it's performance by swapping out activation functions or treating the activation function as a hyperparameter, then you may want to try hand-designed activations like SELU or RL discovered functions like Swish. This tutorial describes these activation functions and others implemented in mxnet in detail.
+
+### Visualizing Activations
+In order to compare the various activation functions and to understand the nuances of their differences we have a snippet of code to to plot the activation functions and their gradients.
```

Comment:
suggestion: ... plot the activation functions (used in the forward pass) and their gradients (used in the backward pass).

Reply:
i like it. thanks!